misc.sk

```diff
@@ -284,9 +284,9 @@ village and pillage update:
 		(many berries|very berried) = -[age=3]
 	{berry age} sweet berry bush [plant] = minecraft:sweet_berry_bush
 
-	flower charge banner pattern¦s = minecraft:flower_banner_pattern
-	creeper charge banner pattern¦s = minecraft:creeper_banner_pattern
-	skull charge banner pattern¦s = minecraft:skull_banner_pattern
+	flower[ charge] banner pattern¦s = minecraft:flower_banner_pattern
+	creeper[ charge] banner pattern¦s = minecraft:creeper_banner_pattern
+	skull[ charge] banner pattern¦s = minecraft:skull_banner_pattern
```

Comment:
You can make the optional part not include the space, Skript makes sure the single space variant is usable anway:
```suggestion
	flower [charge] banner pattern¦s = minecraft:flower_banner_pattern
	creeper [charge] banner pattern¦s = minecraft:creeper_banner_pattern
	skull [charge] banner pattern¦s = minecraft:skull_banner_pattern
```